plugins/modules/zos_mount.py

```diff
@@ -0,0 +1,1118 @@
+    type: str
+    sample: ZFS
+state:
+    description: The desired status of the described mount.
+    returned: always
+    type: str
+    sample: mounted
+persistent:
+    description: Values the user provided as input.
+    returned: always
+    type: dict
+    contains:
+        data_set_name:
+            description: This is the full qualified name of the dataset member to change.
+            returned: always
+            type: str
+            sample: SYS1.FILESYS(BPXPRMAA)
+        backup:
+            description: Specifies whether a backup of destination should be created.
+            returned: always
+            type: bool
+            sample: True
+        backup_name:
+            description: Specify a unique data set name for the destination backup.
+            returned: always
+            type: str
+            sample: SYS1.FILESYS(PRMAABAK)
+tabcomment:
+    description: The text that was used in markers around the Persistent/data_set_name entry.
+    returned: always
+    type: list
+    sample:
+        - [u'I did this because..']
+unmount_opts:
+    description: Describes how the unmount it to be performed.
+    returned: changed and if state=unmounted
+    type: str
+    sample: DRAIN
+mount_opts:
+    description: Options available to the mount.
+    returned: whenever non-None
+    type: str
+    sample: rw,nosecurity
+src_params:
+    description: Specifies a parameter string to be passed to the file system type.
+    returned: whenever non-None
+    type: str
+    sample: D(101)
+tag_untagged:
+    description: Indicates if tags should be written to untagged files.
+    returned: whenever Non-None
+    type: str
+    sample: TEXT
+tag_ccsid:
+    description: CCSID for untagged files in the mounted file system.
+    returned: when tag_untagged is defined
+    type: int
+    sample: 819
+allow_uid:
+    description: Whether the SETUID and SETGID mode bits on executables in this file system are considered.
+    returned: always
+    type: bool
+    sample: yes
+sysname:
+    description: I(sysname) specifies the particular system on which a mount should be performed.
+    returned: if Non-None
+    type: str
+    sample: MVSSYS01
+automove:
+    description:
+        - >
+          Specifies what is to happens to the ownership of a file system during
+          a shutdown, PFS termination, dead system takeover, or file system move occurs.
+    returned: if Non-None
+    type: str
+    sample: AUTOMOVE
+automove_list:
+    description: This specifies the list of servers to include or exclude as destinations.
+    returned: if Non-None
+    type: str
+    sample: I,SERV01,SERV02,SERV03,SERV04
+msg:
+    description: Failure message returned by the module.
+    returned: failure
+    type: str
+    sample: Error while gathering information
+stdout:
+    description: The stdout from the tso mount command.
+    returned: always
+    type: str
+    sample: Copying local file /tmp/foo/src to remote path /tmp/foo/dest.
```

Comment:
Is this correct output from the TSO mount command such that it would say 'copying local file.....` to...

Reply:
doh!  I re-created the line so it is correct.